packages/config/src/common/Milestone.ts

```diff
@@ -1,6 +1,9 @@
+export type MilestoneType = 'general' | 'incident'
+
 export interface Milestone {
   name: string
   link: string
   date: string
   description?: string
+  type?: MilestoneType
```

Comment:
hmm, I am just thinking if it should be optional, probably not

Reply:
This means updating all milestones in the entire codebase, are you sure? :)